http-server/src/main/java/io/airlift/http/server/HttpLogLayout.java

```diff
@@ -37,6 +46,16 @@ public String doLayout(HttpRequestEvent event)
                 .append(event.getTimeToLastByte())
                 .append('\t')
                 .append(event.getTraceToken())
+                .append('\t')
+                .append(event.getProtocolVersion())
+                .append('\t')
+                .append(event.getBeginToDispatchMillis())
+                .append('\t')
+                .append(event.getBeginToEndMillis())
+                .append('\t')
+                .append(event.getFirstToLastContentTimeInMillis())
+                .append('\t')
+                .append(contentInterarrivalStats)
```

Comment:
What are the units for the inter-arrival stats? It's not clear from looking at this method or the summary object.

Reply:
It's in ms, please see `HttpServerChannelListener::processContentTimestamps`. I will add a comment.